packages/genesys-spark-components/src/components/stable/gux-datepicker/gux-datepicker.service.ts

```diff
@@ -187,3 +188,37 @@ export function getIntervalRange(
 
   return { selectionStart, selectionEnd };
 }
+
+export function getFormat(component: HTMLElement): string {
```

Comment:
I think I would move this logic into a function in `sparkIntl` that takes a `locale` not a component. I think that would make it easier to test.
Then this function would look something like 
```
export function getFormat(component: HTMLElement): string {
  const locale = sparkIntl.determineDisplayLocale(component);

  return sparkIntl.determineDisplayLocaleDateFormat(locale); // FYI determineDisplayLocaleDateFormat is a terrible name :)
}
```